levels/lovelace_tower/objectives/api-03-fetch/example.js

```diff
@@ -0,0 +1,9 @@
+async function getMagicalPhrase() {
+  // Web API endpoint: https://twilio.com/quest/magic
+  // TODO: fetch the magical phrase from the endpoint!
+}
+
+(async function () {
+  console.log("Test case data:");
+  console.log((await getMagicalPhrase()).toString());
+})();
```

Comment:
Probably good to avoid the IIFE here in favor of a `runTests` function declaration that we invoke on the next line .